packages/bff/src/services/selfcareService.ts

```diff
@@ -59,6 +76,48 @@ export function selfcareServiceBuilder(
         queries: { userIdForAuth: userId },
       });
     },
+
+    async getInstitutionUsers(
+      tenantId: TenantId,
+      userId: UserId | undefined,
+      query: string | undefined,
+      { authData, logger, headers }: WithLogger<BffAppContext>
+    ): Promise<bffApi.Users> {
+      logger.info(`Retrieving users for institutions ${tenantId}`);
+
+      const roles = authData.userRoles;
```

Comment:
Looking at Scala and the API spec, the roles are not extracted from the auth data, but they are a parameter that comes from the FE

<img width="606" alt="image" src="https://github.com/user-attachments/assets/0c995c4f-4d2e-4269-b865-37052d297408">

Reply:
correct, I see this differences before, my leftover

Reply:
solved in [commit](https://github.com/pagopa/interop-be-monorepo/pull/986/commits/fcd8ac146a590bf51e60f0b27f54441690688066)